lms/templates/header/user_dropdown.html

```diff
@@ -28,11 +28,10 @@
     </a>
 </div>
 <div class="nav-item hidden-mobile nav-item-dropdown" tabindex="-1">
-    <div class="user-dropdown" aria-expanded="false" aria-haspopup="true" aria-controls="user-menu">
-        <span class="sr">user menu</span>
-        <span class="fa fa-caret-down toggle-user-dropdown" aria-hidden="true" tabindex="0"></span>
+    <div class="toggle-user-dropdown" role="button" aria-label="Options Menu" aria-expanded="false" tabindex="0" aria-controls="user-menu">
```

Comment:
String needs to be wrapped for translations.

Reply:
@HarryRein is there a reason that this is a ```div``` with role ```button``` instead of just making it a ```button``` element?

Reply:
Another instance of avoiding the overrides that come with the button class - if you switch in the chrome inspector it has a bunch of things that we would need to add to our styling which made it a bit confusing.